### ISSUE TITLE
[Enhancement]: Add ability to ignore directories

### ISSUE THREAD

**Issue by YMan84** [advplyr/audiobookshelf#516]
### Describe the feature/enhancement

I have the docker library path pointing to my calibre folder, some books of which have audiobooks, some only have an epub. It would be really great if i could add a ".ignore" file to the directory and audiobookshelf would skip it, similiar to how android skips scanning folders for media files if they have a ".ignore" file in the folder. This way i can skip scanning folders that have only an epub file, and only pull in the books that have an audiobook. 

Thanks for the awesome, awesome work!

**Comment by Hoempi** [comment]
I'm running audiobookshelf on a Synology NAS using Docker. Synology Universal search creates a folder "@eaDir" with metadata for it's search engine.

I place a file named .ignore into that directory and re-scanned the item. The files are still shown in the "Files" tab of the item. Did I get the idea behind this wrong?

**Comment by advplyr** [comment]
@Hoempi Can you try scanning the entire library? The library item re-scan may have a bug

**Comment by Hoempi** [comment]
I tried that. I also tried a "Force Re-Scan" of the entire library. Sadly no difference. The current directory structure looks like this:
![grafik](https://user-images.githubusercontent.com/40570297/189284967-6bdbd9df-c293-486a-a754-47036e8c5fe9.png)

Within audiobookshelf I get this:
![grafik](https://user-images.githubusercontent.com/40570297/189285125-46e7c9be-f46c-489b-9530-3a37a6e18531.png)

Should I open a separate bug issue? Are there any logs I could clear and then run the Fore Re-Scan again? Right now I only have this one audiobook in that library anyway, so that wouldn't be a lot of a hussle.

**Comment by advplyr** [comment]
@Hoempi After you scanned in the library item you added the .ignore right?

Can you try removing the library item using the Remove button on the details tab. Then scanning it in again? That will tell us whether it is specific issue to the re-scan logic.

**Comment by Hoempi** [comment]
@advplyr 

Yes, you are right. I only found this issue when searching how to get those files ignored after I saw those files in the details page. Luckily, they are in the tracks or chapters, so it's not that big a problem ;-) But downloading would generate unneeded traffic for those files.

I did as you asked, sadly still no difference. The same if I delete the item and do a Force Re-Scan. I could also try deleting the entire library and adding it again, if that would help you.
